webapp/src/index.tsx

```diff
@@ -16,13 +16,40 @@ export default class Plugin {
     enforceConnectedAccountId = '';
     // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
     public async initialize(registry: PluginRegistry, store: Store<GlobalState, Action<Record<string, unknown>>>) {
-        Client.setServerRoute(getServerRoute(store.getState()));
+        const serverRoute = getServerRoute(store.getState());
+        Client.setServerRoute(serverRoute);
 
         // @see https://developers.mattermost.com/extend/plugins/webapp/reference/
         this.enforceConnectedAccountId = registry.registerRootComponent(EnforceConnectedAccountModal);
 
         registry.registerAdminConsoleCustomSetting('appManifestDownload', MSTeamsAppManifestSetting);
         registry.registerAdminConsoleCustomSetting('ConnectedUsersReportDownload', ListConnectedUsers);
+        registry.registerUserSettings?.({
+            id: manifest.id,
+            icon: `${serverRoute}/plugins/${manifest.id}/public/msteams-sync-icon.svg`,
+            uiName: manifest.name,
+            sections: [{
+                settings: [{
+                    name: 'primary_platform',
+                    options: [
+                        {
+                            text: 'Mattermost',
+                            value: 'mm',
+                            helpText: 'You will get notifications in Mattermost for synced messages and channels. You will need to disable notifications in Microsoft Teams to avoid duplicates. **[Learn more](http://google.com)**',
```

Comment:
@abhijit-singh @johndavidlugtu what plans do we have for this link out to `Learn More`?